web/src/giphy.js

```diff
@@ -115,7 +115,7 @@ async function renderGIPHYGrid(targetEl) {
                     // user to navigate the GIFs using tab.
                     // TODO: Remove this after https://github.com/Giphy/giphy-js/issues/174
                     // is closed.
-                    e.target.tabIndex = 0;
+                    e.target.tabIndex = -1;
                 },
```

Comment:
Why do need this? Also, looks like https://github.com/Giphy/giphy-js/issues/174 is closed, so we can just upgrade our version and remove this.

Reply:
I am not sure how to upgrade the version of giphy-js. Can you guide me a bit on how to do it?

Reply:
You need to update it in `package.json`.

Reply:
@amanagr Updated!